### ISSUE TITLE
OverlayPanel: not opening in Mojarra 2.3

### ISSUE THREAD

**Issue by erickdeoliveiraleal** [primefaces/primefaces#3859]
## 1) Environment
- PrimeFaces version: 6.2
- Does it work on the newest released PrimeFaces version? Version? no
- Does it work on the newest sources in GitHub? no
- Application server + version: wildfly 14
- Affected browsers: all

## 2) Expected behavior

Clicking on the button should open OverlayPanel

## 3) Actual behavior

Not Opening.

## 4) Steps to reproduce

click on the button, the overlay not appears

## 5) Sample XHTML

```
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml"
	xmlns:h="http://java.sun.com/jsf/html"
	xmlns:f="http://java.sun.com/jsf/core"
	xmlns:ui="http://java.sun.com/jsf/facelets"
	xmlns:p="http://primefaces.org/ui"
	xmlns:fn="http://java.sun.com/jsp/jstl/functions">

<h:head>

</h:head>

<h:body>

	<h:form>

		<p:selectOneMenu id="presidio">

			<f:selectItem itemLabel="Nenhum" noSelectionOption="true" />

		</p:selectOneMenu>

	</h:form>

	<h:form id="form">

		<p:commandButton process="@this"
			oncomplete="PF('dlgFolha').show('#{component.clientId}')"
			value="Folha de Ponto" ajax="true" />

	</h:form>

	<p:overlayPanel widgetVar="dlgPdf">
		<h:form id="dlgPdfIn">
			<p:outputPanel>
				<p:panelGrid columns="2" columnClasses="label,value">

					<p:commandButton value="Movimentações" process="@this" />

				</p:panelGrid>
			</p:outputPanel>
		</h:form>
	</p:overlayPanel>
	<p:overlayPanel widgetVar="dlgFolha">
		<h:form id="dlgfolhaIn">
			<p:outputPanel>

				<p:commandButton value="Gerar" ajax="false" />

			</p:outputPanel>
		</h:form>
	</p:overlayPanel>
</h:body>
</html>
```

## 6) Sample bean

not needed

## 7) Working test
https://github.com/erickdeoliveiraleal/primefaces-test/tree/bugselect
[primefaces-test-bugselect.zip](https://github.com/primefaces/primefaces/files/2176665/primefaces-test-bugselect.zip)

**Comment by tandraschko** [comment]
Very likely a Mojarra 2.3 bug.

**Comment by tandraschko** [comment]
As it works fine in all versions, just not in Mojarra 2.3, i pinged Arjan and Bauke.
Maybe you can also create a official issue there.
I will close it for now.

**Comment by erickdeoliveiraleal** [comment]
fixed in #4122
